frontend/src/App.jsx

```diff
@@ -1,9 +1,111 @@
+import { useState } from "react";
 import "./App.css";
 
 function App() {
+  const [numRows, setNumRows] = useState(1);
+  const [numCols, setNumCols] = useState(1);
+  const [matrices, setMatrices] = useState([]);
+
+  const generateMatrix = (e) => {
+    e.preventDefault();
+
+    let data = [];
+    for (let i = 0; i <= numRows; i++) {
```

Comment:
Ei using i and j in loops, you carried the c++ to this project too eer...lol. Unfortunately i don't have access to my emojis here.

Reply:
😆 🤣